### PR TITLE
updated code for the hero feature

### PR DESCRIPTION
**Pull request by cpmortley**

### CODE REVIEW

File: index.htm [hunk 1]
```diff
@@ -22,14 +22,32 @@
       <li><a href="./sites/begonia-details.htm">Shop</a></li>
     </ul>
     <input type="search" id="search-box" name="search" placeholder="Search...">
-    <a href="#" class="button outlined-button">Login</a>
+    <a href="#" class="button outlined-green-button">Login</a>
```
Comment (krutzfeld) [inline index.htm]: its more maintainable when you name that colors: primary, secondary. If you need to change it from green to red then, you just need to change the actual colors in the css not all the names. 

The classes could potentially used in millions of htmls, the css just once.

File: index.htm [hunk 2]
```diff
@@ -22,14 +22,32 @@
...
     <a href="#" class="button filled-button">Sign-up</a>
   </nav>
 
   <div class="wrapper">
-    <h1>WaterMi</h1>
+      <div id="hero-img">
+
```
Comment (krutzfeld) [inline index.htm]: you can remove the empty lines

**Review by krutzfeld — COMMENTED** [review]
Very nice work. I accepted. Please just refactor the minor remarks.

File: index.htm (hunk 1 above)
Comment (cpmortley) [reply index.htm]: Refactoring complete.

File: index.htm (hunk 2 above)
Comment (cpmortley) [reply index.htm]: Refactoring complete.

**Pull request merged**